cephfs/cephfs.go

```diff
@@ -26,15 +31,51 @@ func (e cephError) Error() string {
 	return fmt.Sprintf("cephfs: ret=(%d) %v", e, err)
 }
 
+const (
+	CephStatMask = (C.CEPH_STATX_UID | C.CEPH_STATX_GID | C.CEPH_STATX_SIZE | C.CEPH_STATX_BLOCKS | C.CEPH_STATX_MTIME | C.CEPH_STATX_ATIME)
+)
+
+type CephStat struct {
+	Mode       uint16
+	Uid        uint
+	Gid        uint
+	Size       uint64
+	BlkSize    uint
+	Blocks     uint64
+	AppendTime int64
+	ModifyTime int64
+
+	IsFile    bool // S_ISREG
+	IsDir     bool // S_ISDIR
+	IsSymlink bool // S_ISLNK
+}
+
 // MountInfo exports ceph's ceph_mount_info from libcephfs.cc
 type MountInfo struct {
 	mount *C.struct_ceph_mount_info
 }
 
+type DirResult struct {
+	dirp *C.struct_ceph_dir_result
+}
+
+type Statx struct {
+	stx C.struct_ceph_statx
+}
+
 // CreateMount creates a mount handle for interacting with Ceph.
-func CreateMount() (*MountInfo, error) {
+func CreateMount(id string) (*MountInfo, error) {
```

Comment:
I missed this in the previous review. Existing users may be using the version without a parameter. I'd suggest `CreateMount` and `CreateMountWithClient`